tests/testthat/test-Rules-validity.R

```diff
@@ -880,48 +872,47 @@ test_that("v_stopping_mtd_distribution returns message for non-valid prob", {
 test_that("v_stopping_mtd_cv passes for valid object", {
   object <- StoppingMTDCV(target = 0.3, thresh_cv = 30)
   expect_true(v_stopping_mtd_cv(object))
-})
-
-test_that("v_stopping_mtd_cv returns expected messages for non-valid object", {
-  object <- StoppingMTDCV()
-  object@target <- c(-0.3)
-  object@thresh_cv <- c(-40)
 
-  expect_equal(
-    v_stopping_mtd_cv(object),
-    c(
-      "target must be probability value from (0, 1) interval",
-      "thresh_cv must be percentage > 0"
-    )
-  )
+  object <- StoppingMTDCV(target = 0.3, thresh_cv = 100)
+  expect_true(v_stopping_mtd_cv(object))
 })
 
-test_that("v_stopping_mtd_cv returns expected messages for non-valid object", {
-  object <- StoppingMTDCV()
-  object@target <- c(3)
-  object@thresh_cv <- c(0)
+test_that("v_stopping_mtd_cv returns message for non-valid target", {
+  err_msg <- "target must be probability value from (0, 1) interval"
+  object <- StoppingMTDCV(target = 0.3, thresh_cv = 30)
 
-  expect_equal(
-    v_stopping_mtd_cv(object),
-    c(
-      "target must be probability value from (0, 1) interval",
-      "thresh_cv must be percentage > 0"
-    )
-  )
+  # Changing `target` so that it does not represent allowed probability value.
+  object@target <- 1
```

Comment:
I thought that should be ok?

Reply:
See the comment for the first conversation, above.